modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,72 @@
+= Multi-factor Authentication
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Multi-factor authentication provides an additional layer of verification for individual Anypoint Platform users.
+You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings.
+
+When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using one of the following authentication methods:
+
+* One-time password generator
+* Built-in authenticator
+* Security key
+* Salesforce Authenticator
+
+// will we let orgs require users to configure only certain methods? Or multiple methods?
+
+== Multi-factor Authentication Configurations
+
+You can configure the following verification methods to comply with multi-factor authentication requirements:
+
+One-time password generator:: Registers an authenticator app to create verification codes that you provide when logging in to Anypoint Platform.
+
+Built-in authenticator:: Registers a physical authentication device, such as Touch ID, to verify your identity when logging in to Anypoint Platform.
+
+// what happens if they lose this device, and it's their only MFA method?...
+
+Security key:: Registers a USB security key to your account. The device is then authorized to create verification codes that you provide when logging in to Anypoint Platform.
+
+Salesforce Authenticator:: Registers the Salesforce Authenticator mobile app to create verification codes that you provide when logging in to Anypoint Platform.
+
+== Enable Multi-factor Authentication for All User Accounts
```

Comment:
Can we also get a section on what happens when MFA for the org is disabled? (Users with MFA configured will still be prompted, but can unenroll from their profile)

Reply:
Yep.

Reply:
Added along with steps to disable MFA requirements.